web/src/components/organisms/Project/Content/ContentList/hooks.ts

```diff
@@ -149,6 +150,51 @@ export default () => {
     }
   });
 
+  const [updateItemMutation] = useUpdateItemMutation();
```

Comment:
You can add refetch queries:
```suggestion
  const [updateItemMutation] = useUpdateItemMutation({
    refetchQueries: ["SearchItem", "GetViews"],
  });
```